docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,386 @@
+    httpd = server_class((HOST_NAME, PORT_NUMBER), MyHandler)
+    print(time.asctime(), "Server Starts - {}:{}".format(HOST_NAME, PORT_NUMBER))
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        pass
+    httpd.server_close()
+    print(time.asctime(), "Server Stops - {}:{}".format(HOST_NAME, PORT_NUMBER))
+```
+
+The code snippet simply starts a Python server and serves the get requests with HTML that says the current time. You should be able to run this code snippet with `python3 helloworld.py` and browse [localhost:8000](http://localhost:8000).
+
+#### Change port mapping to be dynamic
+
+If we want to get a port number dynamically from available ports, Marathon provides a way to achieve this. We access the available ports using the environment variable `$PORT0`, `$PORT1`, `$PORT2` and so on. This will be explained clearly in the later section. But as of now, just change your Python snippet to read the port from an environment variable as below :
+
+```python
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+```
+
+Once you do this, you can browse your server by executing `PORT0=8000 python3 helloworld.py`
+
+### Step 2 : Creating a Docker container
+Creating a Docker container is essential to distribute your service. It runs completely isolated from the host environment by default, only accessing host files and ports if configured to do so. To continue reading, you need to be familiar with docker. We recommend this [get-started](https://docs.docker.com/get-started/) guide to familiarize docker. You should have logged in to your Docker account in your terminal using `docker login`.
+
+We create a Docker file (named `Dockerfile`) in the `time-server-service` directory created earlier. The `Dockerfile` should look like this:
+
+```
+# Use an official Python runtime as a base image
+FROM python:3
+
+# Set the working directory to the package directory
+WORKDIR /package
+
+# Copy the current directory contents into the container at /package
+ADD . /package
+
+# Run helloworld.py when the container launches. -u flag makes sure the output is not buffered
+CMD ["python3", "-u", "helloworld.py"]
+```
+Read through the comments to understand what each line in the `Dockerfile` does.
+
+
+#### Build the container
+*Through out the rest of this guide, we refer to `docker-user-name` as your Docker user name where you access your Docker images. You are expected to replace the keyword `docker-user-name` with your Docker user name in all commands and files*
+Now that we have everything ready, we can build our container. Here’s what `ls` should show:
+```
+$ ls
+Dockerfile		helloworld.py
+```
+Now run the build command. This creates a Docker image which we’re going to tag using -t so it has a friendly name.
+
+`docker build -t docker-user-name/time-server:part1 .`
+
+Where is your built image? It’s in your machine’s local Docker image registry:
```

Comment:
I think "It's in your machine's local Docker daemon" might be a more accurate statement.